src/loader-post.js

```diff
@@ -9,7 +9,7 @@ export default class Loader extends Component {
 
     console.log(loadContext)
     const customLoader = loadContext.loaders.Post
```

Comment:
I think we need to detect whether `loaders` have been declared on `loadContext` before we access `Post`. If you remove `loaders` from `tapestry.js` config, it'll break. My bad. 🐙